app/src/renderer/components/wallet/PageWallet.vue

```diff
@@ -12,8 +12,8 @@
     <tm-data-msg id="account_empty_msg" slot="no-data" icon="help_outline">
       <div slot="title">Account empty</div>
       <div slot="subtitle">
-        This account doesn't hold any coins yet. Go to the&nbsp;
-        <a href="https://gaia.faucetcosmos.network/">token faucet</a> &nbsp;to
+        This account doesn't hold any coins yet. Visit the
+        <a href="https://gaia.faucetcosmos.network/">token&nbsp;faucet</a> to
```

Comment:
hmmm I'd remove the second sentence, we don't want to send users to links that don't work at all